src/ontology/mondo-edit.obo

```diff
@@ -570175,6 +570185,3049 @@ intersection_of: MONDO:0700097 MONDO:0000721 ! cross-species analog xanthinuria
+xref: OMIA:002398 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002398"} ! non-human animal disease
+is_a: MONDO:1011360 {source="https://orcid.org/0000-0002-5002-8648"} ! syndromic disease, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011903
+name: renal amyloidosis, non-human animal
+xref: OMIA:000040 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000040"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000040", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011904
+name: chronic interstitial nephropathy, non-human animal
+xref: OMIA:000192 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000192"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000192", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011905
+name: glomerulonephropathy, non-human animal
+xref: OMIA:000414 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000414"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000414", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011906
+name: hemolytic uremic syndrome, non-human animal
+xref: OMIA:000435 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000435"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000435", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011907
+name: nephropathy, non-human animal
+xref: OMIA:000710 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000710"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000710", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011908
+name: polycystic mononephrosis, non-human animal
+xref: OMIA:000808 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000808"} ! non-human animal disease
+is_a: MONDO:1011365 {source="OMIA:000808", source="https://orcid.org/0000-0002-5002-8648"} ! urinary system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011909
+name: renal disease, non-human animal
```

Comment:
shouldn't we have a x-species for this one?

Reply:
added